frb_example/benches/dart/lib/env/stub.dart

```diff
@@ -0,0 +1,3 @@
+// see https://github.com/flutter/flutter/issues/55870
```

Comment:
uber-nit: this comment seems unrelated to stub.dart

Reply:
I mean, my intent was that, if any env related bugs show up again, people don't lose hours figuring out (especially when there's no error at runtime, it's just that your env var is silently ignored) and get directly into the heart of the matter.

Reply:
Anyway this is not a big thing :)